src/wp-includes/media.php

```diff
@@ -1843,6 +1843,11 @@ function wp_filter_content_tags( $content, $context = null ) {
 				$filtered_image = wp_img_tag_add_loading_attr( $filtered_image, $context );
 			}
 
+			// Use alternate mime types when specified and available.
+			if ( $attachment_id > 0 ) {
+				$filtered_image = wp_image_use_alternate_mime_types( $filtered_image, $attachment_id );
```

Comment:
This function should also receive the `$context` as second parameter, similar to e.g. `wp_img_tag_add_width_and_height_attr()` above. `$attachment_id` can become the third parameter.

Reply:
added context to function, call and filter